src/use-lunatic/__snapshots__/use-lunatic.test.ts.snap

```diff
@@ -1855,329 +1855,6 @@ exports[`use-lunatic() > overview > with loop > should handle initialPage 1`] =
 ]
 `;
 
-exports[`use-lunatic() > overview > with loop > should handle lastReachedPage 1`] = `
```

Comment:
Why does it not work anymore?

Reply:
It was an outaded test. (vitest throw a warn)
Nothing is broken.

It's a older than this PR.